### PR TITLE
Scheduler dockerfile; updating mesos -> 0.25.0

### PR DESCRIPTION
**Pull request by sashkachan**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+RUN echo "deb http://ppa.launchpad.net/openjdk-r/ppa/ubuntu trusty main" > /etc/apt/sources.list.d/openjdk.list && \
+apt-key adv --keyserver keyserver.ubuntu.com --recv 86F44E2A && \
+apt-get update && \
+apt-get -y install openjdk-8-jre-headless && \
+rm -rf /var/lib/apt/lists/*
+
+RUN echo "deb http://repos.mesosphere.io/ubuntu/ trusty main" > /etc/apt/sources.list.d/mesosphere.list && \
+apt-key adv --keyserver keyserver.ubuntu.com --recv E56151BF && \
+apt-get -y update && \
+apt-get -y install mesos=0.25.0-0.2.70.ubuntu1404 && \
```
Comment (sadovnikov) [inline Dockerfile:12]: What do we need Mesos in Kibana container for? 
What is this new container for?

Comment (sashkachan) [inline Dockerfile:12]: That's for libmesos.
The long-term intention is to ELK frameworks operate with similar infrastructure:
- support jar
- support Docker
- similar distribution and build scripts

Perhaps I should have added it in a similar branch.

Comment (sadovnikov) [inline Dockerfile:12]: No additional tasks are to be executed in container with Kibana, it's not a Mesos Agent

**Comment by sadovnikov** [comment]
No additional tasks are expected to run in Kibana container, it's not a Mesos Agent

File: Dockerfile (hunk 1 above)
Comment (sashkachan) [inline Dockerfile:12]: > No additional tasks are to be executed in container with Kibana, it's not a Mesos Agent

I don't understand the point you are making. libmesos is required to start the framework. It's nothing to do with being a mesos agent. 
Anyway, it's not the right branch for that.

**Pull request closed without merging**